src/sql/workbench/browser/modal/modal.ts

```diff
@@ -412,6 +412,24 @@ export abstract class Modal extends Disposable implements IThemable {
 		});
 	}
 
+	/**
+	* removes the footer button matching the provided label
+	* @param label Label on the button
+	*/
+	protected removeFooterButton(label: string): void {
+		let buttonIndex = this._footerButtons.findIndex(e => {
+			try {
+				return e && e.element.innerText === label;
+			} catch {
+				return false;
+			}
+		});
+		let button = this._footerButtons[buttonIndex];
```

Comment:
Consider adding a check for findIndex not finding the item (buttonIndex === -1) (or undefined if you switch to using find directly) that skips the removal if we couldn't find the item

Reply:
ah yes good catch.. will do